Fix configure failure for aarch64

| -- Performing Test CXX_COMPILER_SUPPORTS_-fcolor-diagnostics - Failed
| -- Performing Test CXX_COMPILER_SUPPORTS_-fdiagnostics-color=always
| -- Performing Test CXX_COMPILER_SUPPORTS_-fdiagnostics-color=always - Success
| CMake Error at Source/cmake/OptionsCommon.cmake:42 (WEBKIT_PREPEND_GLOBAL_COMPILER_FLAG):
|   Unknown CMake command "WEBKIT_PREPEND_GLOBAL_COMPILER_FLAG".
| Call Stack (most recent call first):
|   Source/cmake/WebKitCommon.cmake:58 (include)
|   CMakeLists.txt:166 (include)
|

Upstream-Status: Pending

Signed-off-by: Manjukumar Matha <manjukumar.harthikote-matha@xilinx.com>

Index: webkitgtk-2.14.6/Source/cmake/OptionsCommon.cmake
===================================================================
--- webkitgtk-2.14.6.orig/Source/cmake/OptionsCommon.cmake     2018-02-15 18:56:29.315238564 -0800
+++ webkitgtk-2.14.6/Source/cmake/OptionsCommon.cmake  2018-02-15 18:54:57.469518064 -0800
@@ -39,7 +39,7 @@
     if (NOT WTF_CPU_ARM64)
         message(FATAL_ERROR "WTF_CPU_ARM64_CORTEXA53 set without WTF_CPU_ARM64")
     endif ()
-    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAG(-mfix-cortex-a53-835769)
+    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-mfix-cortex-a53-835769)
 endif ()

 EXPOSE_VARIABLE_TO_BUILD(WTF_CPU_ARM64_CORTEXA53)
